server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/SearchActionTest.java

```diff
@@ -128,21 +128,23 @@ public void search_map_dates() {
     qualityProfileDb.insertQualityProfiles(newQualityProfileDto()
       .setLanguage(xoo1.getKey())
       .setRulesUpdatedAt("2016-12-21T19:10:03+0100")
-      .setLastUsed(time));
+      .setLastUsed(time)
+      .setUserUpdatedAt(time));
 
     SearchWsResponse result = call(ws.newRequest());
 
     assertThat(result.getProfilesCount()).isEqualTo(1);
     assertThat(result.getProfiles(0).getRulesUpdatedAt()).isEqualTo("2016-12-21T19:10:03+0100");
     assertThat(parseDateTime(result.getProfiles(0).getLastUsed()).getTime()).isEqualTo(time);
+    assertThat(parseDateTime(result.getProfiles(0).getUserUpdatedAt()).getTime()).isEqualTo(time);
```

Comment:
You should add a test (or a check in another test) that userUpdatedAt is not returned when the profile was not manually updated